weave/flow/eval.py

```diff
@@ -397,7 +420,7 @@ async def predict_and_score(
             scores[scorer_name] = result
 
         return {
-            "output": model_output,
+            self._output_key: model_output,
```

Comment:
this should always be "output" even if you have some scorers that use "model_output"